website/src/pages/hackathons.tsx

```diff
@@ -0,0 +1,94 @@
+                    eventLogger.trackHackathonFormSubmitted()
+                },
+            })
+        })
+    }
+
+    public render(): JSX.Element | null {
+        return (
+            <Layout location={this.props.location}>
+                <div>
+                    <Helmet>
+                        <title>Sourcegraph - Set up Sourcegraph at a Hackathon</title>
+                        <meta name="twitter:title" content="Set up Sourcegraph at a Hackathon" />
+                        <meta property="og:title" content="Set up Sourcegraph at a Hackathon" />
+                    </Helmet>
+                    <div className="pricing-bg" />
+                    <div className="jumbotron text-right dark-7">
+                        <div className="row">
+                            <div className="sales py-0 d-flex flex-grow flex-column flex-lg-row align-items-center justify-content-around">
+                                <div>
+                                    <h1 className="text-center">Set up Sourcegraph during a hackathon</h1>
+                                    <p className="text-center measure">
+                                        Hackathons are the perfect time to set up code search and intelligence for your
+                                        team. We'll give you access to all of Sourcegraph's features to set you up for
+                                        the best possible demo.
+                                    </p>
+                                </div>
+                            </div>
+                        </div>
+                    </div>
+                    <section className="d-flex justify-content-around flex-column flex-lg-row sales bg-white">
+                        <div className="panel-area mr-lg-5 mt-0 mb-2">
+                            <div className="d-flex flex-column panel">
+                                <p>
+                                    Want to bring lasting value to your company during a hackathon? Set up Sourcegraph
+                                    and bring the power of code search and code intelligence to your engineering team.
+                                </p>
+                                <ul>
+                                    <li>
+                                        We'll give you <b>access to our Enterprise features</b> so you can ship
+                                        Sourcegraph's full feature set to your team and for your demo.
+                                    </li>
+                                    <li>
+                                        We'll <b>answer any questions and share tips</b> to make sure you can
+                                        successfully deploy and demo Sourcegraph
+                                    </li>
+                                </ul>
+
+                                <p>Fill out the form, and we'll get back to you ASAP on how to get started!</p>
```

Comment:
How about "fill out the form or Tweet at us" to make it a bit less formal? This page simply existing is the main thing that'll help :)